docs/index.md

```diff
@@ -14,7 +14,7 @@ first_example
 
 ```{toctree}
 ---
-caption: Overview
+caption: API Functionalities
```

Comment:
I am not sure if Functionalities is the best word here.
I understand the Overview is maybe too generic, and needs to be changed.
But Functionalities is a lot about what it does, and this is more about how it works.
WDYT?

Reply:
Agreed here, I also thought functionalities was to specific to the methods and exactly to what it does. 
I think Overview is still fine since these sets the stage for the more nitty gritty details.

Reply:
I get you, how about any of these?
- api workflow
- api architecture 
- usage

Reply:
What do the readthedocs of similar systems usually show?
If we don't find something really strong, I think it's fine if we keep overview for now...

Reply:
Usually what I found as best practice:

- capabilities
- User guide/ api usage
- features

Overview is too broad, and confusing because I feel we have so much intros now. In this part of the doc, we are in the specificities and we need to make that explicit for the reader so they don't get frustrated when navigating the overall doc 

I'll send you some examples to the chat

Reply:
OK! I see.
But it's not about capabilities, functionalities (what it does), or features (what it has). 
It could be something like "how it works". It's about allowing users to build  a mental image of what is going on, and why things happen in certain ways. 

"Understanding the API"?

Reply:
I love it! I was worried we were passing on using the verbs. this is quite cool and it would flow well with the rest of the stuff. I'll adjust stuff.